### ISSUE TITLE
NTR, pulmo: Pulmonary Bullae

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5575]
**Preferred term label:**
Pulmonary Bullae

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pulmonary Bullae are rounded focal regions of emphysema with a thin wall which measure more than 1 cm in diameter. They are often subpleural in location and are typically larger in the apices. In some cases, bullae can be very large and result in compression of adjacent lung tissue. A giant bulla is arbitrarily defined as one that occupies at least one third of the volume of a hemithorax. When large, bullae can simulate pneumothorax. The most common cause is paraseptal emphysema but bullae may also be seen in association with centrilobular emphysema.

PMID: 18195376

**Parent term (use hpo.jax.org/app)**
Emphysema HP:0002097

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
added term
